packages/hw-app-elrond/src/Elrond.js

```diff
@@ -0,0 +1,208 @@
+//@flow
+
+import type Transport from "@ledgerhq/hw-transport";
+import BIPPath from "bip32-path";
+
+const CHUNK_SIZE = 150;
+const CURVE_MASK = 0x80;
+const CLA = 0xed;
+
+const INS = {
+  GET_VERSION: 0x02,
```

Comment:
Shouldn't `GET_VERSION` be `0x01`, according to https://github.com/ElrondNetwork/ledger-elrond/blob/master/src/main.c#L28? (maybe I get it wrong)